concordium-consensus/src/Concordium/Birk/Bake.hs

```diff
@@ -39,7 +39,7 @@ import Concordium.GlobalState.Parameters
 import Concordium.GlobalState.TransactionTable
 import Concordium.GlobalState.TreeState as TS
 import Concordium.Types.HashableTo
-import Concordium.Types.SeedState
+import Concordium.Types.SeedState hiding (getSeedState)
```

Comment:
Why hide this?

Reply:
It collides with `getSeedState` from `BlockStateQuery`.s

Reply:
I've renamed the `getSeedState` to `deserializeSeedState` to avoid this collision.